cmd/memresolverconfigbuilder_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2018-2019 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/nats-io/nsc/cmd/store"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_MemResolverContainsStandardProperties(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+
+	builder := NewMemResolverConfigBuilder()
+	a, err := ts.Store.Read(store.Accounts, "A", store.JwtName("A"))
+	require.NoError(t, err)
+	err = builder.Add(a)
+	require.NoError(t, err)
+
+	d, err := builder.Generate("")
+	require.NoError(t, err)
+
+	conf := string(d)
+	require.Contains(t, conf, "resolver: MEMORY")
+	require.Contains(t, conf, "resolver_preload: {")
```

Comment:
Maybe have the server code parse it correctly and look at the opts it generates?

Reply:
yes added a test for this, in this case I am testing that my code is generating what I expect to be there.

Reply:
I mean actually have the server's parser parse the config for validity.

Reply:
yes - I added a test, see last comment.